sdcm/tester.py

```diff
@@ -300,7 +300,15 @@ def __init__(self, *args, **kwargs):  # pylint: disable=too-many-statements,too-
         self.log.debug("IP used for SSH connections is '%s'",
                        ip_ssh_connections)
         self.test_config.set_ip_ssh_connections(ip_ssh_connections)
-        self._duration = self.params.get(key='test_duration')
+
+        self._stress_duration: int = self.params.get('stress_duration')
```

Comment:
will it be `int` if `stress_duration` is not set at all? Shouldn't we set default to 0 in sct_configuration?

Reply:
yes, in this case it will taken from test_default which is 0 by default.